pennylane/transforms/insert_ops.py

```diff
@@ -235,34 +232,49 @@ def f(w, x, y, z):
 
     if not isinstance(op_args, Sequence):
         op_args = [op_args]
-
-    for prep_op in circuit.operations[: circuit.num_preps]:
-        apply(prep_op)
+    new_operations = []
+    for prep_op in tape.operations[: tape.num_preps]:
+        new_operations.append(prep_op)
 
     if position == "start":
-        for w in circuit.wires:
-            op(*op_args, wires=w)
+        for w in tape.wires:
+            with qml.QueuingManager.stop_recording():
```

Comment:
I don't think this is necessary, nothing should get queued to surrounding contexts if done within `make_qscript`. same for the rest of this file

Reply:
one of those better safe than sorry things.  operations being queued from random places can be rather hard to track down, as we discovered recently with those qnode tests.

Reply:
I mean, it's not the biggest deal but AnnotatedQueue's don't themselves queue so this is unnecessary overhead.
```pycon
>>> def f():
...         qml.PauliX(0)
...
>>> with qml.queuing.AnnotatedQueue() as q:
...         qs = qml.tape.make_qscript(f)()
...
>>> q.queue, qs.operations
([], [PauliX(wires=[0])])
```

Reply:
Let me give a try